frontend/www/js/omegaup/components/arena/Scoreboard.vue

```diff
@@ -178,35 +178,38 @@ export default class ArenaScoreboard extends Vue {
     padding: 0.2em;
     .points {
       font-weight: bold;
-      width: 3.5em;
-      border-right-style: dotted;
     }
     .penalty {
       font-size: 70%;
-      border-left-width: 0;
-    }
-    .accepted {
-      background: #dfd;
-    }
-    .pending {
-      background: #ddf;
-    }
-    .wrong {
-      background: #fdd;
     }
-    .position.recent-event {
-      font-weight: bold;
-      background: #dfd;
-    }
-    .accepted.recent-event {
-      background: #8f8;
-    }
-    .position {
-      width: 3.5em;
+    div[class$='points'] {
```

Comment:
```suggestion
    div.points {
```

esto funciona? si no, por qué no funciona? y una vez que se identifique, hay que ver cómo hacer que funcione.

igual abajo.

Reply:
Ahorita quité todo lo referente a estos estilos hasta conocer en cuáles casos si se deben pintar y cuáles no.